### ISSUE TITLE
Problems with Sequent Display

### ISSUE THREAD

**Issue by davidfaraci** [gleachkr/Carnap-Old#35]
Sequents do not display premises in stated order. Example:

![image](https://cloud.githubusercontent.com/assets/13985235/9504192/340b2320-4c09-11e5-84ec-3c8b8843a3c3.png)
Sequents sometimes display premises multiple times. Example:

![image](https://cloud.githubusercontent.com/assets/13985235/9504154/009b9240-4c09-11e5-88a1-d5ed0dfd61a0.png)

**Comment by gleachkr** [comment]
Ok, I think I've fixed the repeated premise issue. The ordering of premises is tricky---it's determined basically by the form of the inference rules you use, I think, which can interact in complicated ways pretty quickly. 

I think the easiest way to get the order of premises in the sequent to match the order of their occurrence in the proof would be to look the proof after figuring out what it established, and then to re-order the premises. This would involve a lot of code reorganization. I think it might be better to put this on hold until writing a new natural deduction module like the one projected in #23.

**Comment by davidfaraci** [comment]
Repeated premise issues seems to be fixed. Sounds like a good idea to wait on the other issue.

**Comment by gleachkr** [comment]
Ok. I'll add a note to #23
